admin/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="robots" content="noindex" />
+    <title>Content Manager</title>
+
+    <!-- fix mobile responsiveness -->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/hithismani/responsive-decap@main/dist/responsive.css">
```

Comment:
This is an extra I've used before. 
`dark.css` can be removed in you don't care to support dark mode or it's broken. Everything else is just copy/pasted from the Decap docs. This is all this file needs indefinitely; the decap js script does the rest.